includes/data-port/models/class-sensei-data-port-question-model.php

```diff
@@ -128,26 +448,60 @@ public static function get_schema() {
 				'type' => 'string',
 			],
 			self::COLUMN_FEEDBACK        => [
-				'type' => 'string',
+				'type'       => 'string',
+				'allow_html' => true,
 			],
 			self::COLUMN_TEXT_BEFORE_GAP => [
-				'type' => 'string',
+				'type'       => 'string',
+				'validator'  => static::validate_for_question_type( 'gap-fill', false ),
```

Comment:
I like the idea of this validations. We should make sure that we document the following cases when we release though:
1. Insert will fail if type == gap-fill and these three columns are missing.
2. Insert will fail if type != gap-fill and these columns exist.